src/app/hooks/reactQuery.ts

```diff
@@ -70,13 +130,41 @@ export function useProgramTypeOfYear(
   return useQuery({
     queryKey: ["programs", programType, catalogYear],
     queryFn: async () => {
-      console.log("fetching programs", programType, catalogYear);
       return await getProgramsByTypeInYear(programType, catalogYear);
     },
     placeholderData: [],
     enabled: catalogYear !== "" && programType in ProgramType,
   });
 }
+
+/**
+ * A React Query hook to fetch all catalog years, optionally for a specific
+ * program
+ * @param programName (Optional) Name of the program
+ * @returns React Query useQuery Hook for all catalog years
+ */
+export function useCatalogYears() {
```

Comment:
This is a little redundant having two hooks with very similar implementations

Reply:
which part exactly is redundant, `useCatalogYears` and `useCatalogYearsForProgram`? I was thinking it was clearer to have different functions for some reason but I think you're right it makes sense to combine and have `programName` as an optional.

Reply:
I used this pattern for a bunch of hooks I can undo them all

Reply:
Might be better to merge all "redundant" hooks especially if they use the same underlying server actions